pkgs/applications/graphics/inkscape/extensions/textext/default.nix

```diff
@@ -58,6 +58,8 @@ python3.pkgs.buildPythonApplication rec {
     python3.pkgs.lxml
     python3.pkgs.cssselect
     python3.pkgs.numpy
+
+    python3.pkgs.pygobject3
```

Comment:
That is already present on line 57.

Reply:
You're right. I have no idea how I missed it. And more importantly, I have no idea why it wasn't working before and now started working with. I will close this PR.